scripts/check-security

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# This script checks if there is plain (not redacted) log in TiKV.
+
+set -e
+
+if grep -r --color=always -n -C 1 hex::encode_upper {components,src}/ ; then
```

Comment:
Let's filter `encode_upper` (without `hex::` prefix) instead. Let's be stricter.